macros/g4simulations/G4_Tracking.C

```diff
@@ -82,6 +89,10 @@ int n_tpc_layer_mid = 16;
 int n_tpc_layer_outer = 16;
 int n_gas_layer = n_tpc_layer_inner + n_tpc_layer_mid + n_tpc_layer_outer;
 
+///////////////// Micromegas
+bool enable_micromegas = false;
+const int n_micromegas_layer = 2;
```

Comment:
micromegas would contribute to the tracking and the momentum resolution. Shall we default it as `n_micromegas_layer = 0`, so the default behavior do not change?

Reply:
Ignore my question. Missed the line above it....